drivers/slipdev/Kconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_SLIPDEV
+    bool "Configure SLIPDEV driver"
+    depends on MODULE_SLIPDEV
+    help
+        Configure the SLIPDEV driver using Kconfig.
+
+if KCONFIG_MODULE_SLIPDEV
+
+choice
+    bool "Buffer Size"
+    default SLIPDEV_BUFSIZE_2048
+    help
+        UART buffer size used for TX and RX buffers.
+        Reduce this value if your expected traffic does
+        not include full IPv6 MTU.
+        Value needs to be power of two and `<= INT_MAX`
+
+config SLIPDEV_BUFSIZE_512
+    bool "512"
+
+config SLIPDEV_BUFSIZE_1024
+    bool "1024"
+
+config SLIPDEV_BUFSIZE_2048
+    bool "2048"
+
+config SLIPDEV_BUFSIZE_CUSTOM
+    bool "Custom"
+
+config SLIPDEV_BUFSIZE
+    int "Custom"
+    default 2048
+    depends on SLIPDEV_BUFSIZE_CUSTOM
+
+endchoice
```

Comment:
What if this is modeled as the exponent of `2^n`? Similar to https://github.com/RIOT-OS/RIOT/blob/4ceff67ca0b1e88abf8bdcb5e854d89146428c92/sys/usb/usbus/cdc/acm/Kconfig#L15-L23

That way it's easy to impose the power-of-two and maximum value restrictions.

Reply:
Done. I have added the option for custom. Retained some macro for better user experience.

Reply:
I think having the choice complicates the logic here. I would just keep the exponent.

Reply:
Done. Updated and tested.